arwendebug/database.go

```diff
@@ -0,0 +1,112 @@
+package arwendebug
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+)
+
+type database struct {
+	rootPath string
+}
+
+// newDatabase creates a new debugging database (basically, a folder with JSON files)
+func newDatabase(rootPath string) *database {
+	database := &database{rootPath: rootPath}
```

Comment:
Maybe use `db`? There are some warnings for the conflicting name with the struct name

Reply:
Renamed.